_includes/patterns-filter.html

```diff
@@ -361,9 +361,9 @@
 
     // If no checkbox is selected, show all pattern cards and enable the "All" radio button
     if (!filterArray.length) {
-      for (let j = 0; j < patternCards.length; j++) {
-        patternCards[j].style.display = "block";
-      }
+      // for (let j = 0; j < patternCards.length; j++) {
```

Comment:
why commented this?

Reply:
The loop was displaying all the patternCards on the 1st load.